tests/network/hotplug.go

```diff
@@ -43,13 +43,6 @@ const (
 	nadName   = "skynet"
 )
 
-type hotplugMethod string
-
-const (
-	migrationBased hotplugMethod = "migrationBased"
-	inPlace        hotplugMethod = "inPlace"
```

Comment:
Before this commit, `verifySriovDynamicInterfaceChange` could receive `plugMethod`.
This commit drops the above parameter, which only received `migrationBased` as its argument.

And the reason for it, AFAIU, is that hotplug of an SR-IOV  NIC is only applicable using the migration based approach (due to the fact it needs proper scheduling and assistance from the device plugin).

Reply:
> The reason for dropping this parameter is absent from the commit message. I find this minimal explanation to be beneficial. Please consider adding it.

The reason is syntactic and trivial, I did not know we need to be that pedantic to explain simple stuff like this.

Reply:
>  my feedback was to give an explicit reasoning - which could be as simple as "the removed parameter always received a single value so it was dropped"

I guess the disagreement is about what deserves reasoning and what does not require it due to it being trivial. As the author, I expected the reader to understand why it was gone by reading the code changes and it was a side-effect of implementing what is mentioned in the commit.

Reply:
If the commit was focused on removing the parameter, I agree with you.
But since it is a side affect of a bigger change, I don't think its fair to assume that reviewers will catch it.